Lesson-5/lesson/src/Components/Counter/Counter.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+export default function Counter() {
+  const [count, setCount] = useState(0);
+
+  const handleCountUp = () => {
+    setCount(count + 1);
+  };
+  
+  const handleCountDown = () => {
+    if (count <= 1) setCount(0);
+    else setCount(count - 1);
```

Comment:
mình nên bọc đoạn code if, else vào trong cắp dấu {} cho rõ ràng,
sau này nếu mình có thêm code gì đó trong phần if hoặc else thì cũng dễ dàng

```suggestion
    if (count <= 1) {
         setCount(0);
    } else {
         setCount(count - 1);
    }
```

Reply:
Dạ oke ạ, bình thường em vẫn viết viết đủ dấu ngoặc nhưng cảm giác viết React cứ muốn viết tắt ấy ạ. Để em tập quen dần